projects/example_project/README.md

```diff
@@ -21,27 +26,123 @@ This project implements a dummy MAE wrapper, which prints "Welcome to MMSelfSup"
 
 ## Usage
 
-<!-- For a typical model, this section should contain the commands for training and testing. You are also suggested to dump your environment specification to env.yml by `conda env export > env.yml`. -->
+<!-- For a typical model, this section should contain the commands for dataset prepareation, pre-training, downstream tasks. You are also suggested to dump your environment specification to env.yml by `conda env export > env.yml`. -->
 
-### Pre-training commands
+### Setup Environment
 
-In MMSelfSup's root directory, run the following command to train the model:
+It requires [PyTorch](https://pytorch.org/get-started/locally/) and the following OpenMMLab packages:
+
+- [MIM](https://github.com/open-mmlab/mim): A command-line tool to manage OpenMMLab packages and experiments.
+- [MMEngine](https://github.com/open-mmlab/mmengine): OpenMMLab foundational library for training deep learning models.
+- [MMCV](https://github.com/open-mmlab/mmcv): OpenMMLab foundational library for computer vision.
+- [MMClassification](https://github.com/open-mmlab/mmclassification): OpenMMLab image classification toolbox and benchmark. Besides classification, it's also a repository to store various backbones.
+
+Assume you have prepared your Python and PyTorch environment, just use the following command to setup the environment.
 
 ```bash
-python tools/train.py projects/example_project/configs/dummy-mae_vit-base-p16_8xb512-amp-coslr-300e_in1k.py
+pip install openmim
+mim install mmengine "mmcv>=2.0.0rc1"
+mim install mmselfsup
 ```
 
-### Downstream tasks commands
+### Data Preparation
 
-In MMSelfSup's root directory, run the following command to train the downstream model:
+To show the dataset directory or provide the commands for dataset preparation if needed.
+
+For example:
+
+```text
+data/
+└── imagenet
+    ├── train
+    ├── val
+    └── meta
+        ├── train.txt
+        └── val.txt
+```
+
+### Pre-training Commands
+
+At first, you need to add the current folder the the `PYTHONPATH`, so that Python can find your model files. In `example_project/` root directory, please ;run command below to add it.
```

Comment:
```suggestion
At first, you need to add the current folder the the `PYTHONPATH`, so that Python can find your model files. In `example_project/` root directory, please run command below to add it.
```